vnext/Shared/HermesRuntimeHolder.cpp

```diff
@@ -132,5 +132,33 @@ void HermesRuntimeHolder::initRuntime() noexcept {
   errorPrototype.setProperty(*m_hermesRuntime, "jsEngine", "hermes");
 }
 
+#ifdef HERMES_ENABLE_DEBUGGER
+
+std::string HermesRuntimeHolder::getDebugTargetName(const DevSettings& devSettings)  noexcept {
```

Comment:
This logic should probably be shared across the VS engines, not be in a Hermes specific location.